qa/L0_backend_python/env/test.sh

```diff
@@ -120,6 +120,38 @@ for EXPECTED_VERSION_STRING in "$PY36_VERSION_STRING" "$PY37_VERSION_STRING" "$P
 done
 set -e
 
+rm $SERVER_LOG
+
+## Test re-extraction of environment.
+SERVER_ARGS="--model-repository=`pwd`/models --log-verbose=1 --model-control-mode=explicit"
+run_server
+if [ "$SERVER_PID" == "0" ]; then
+    echo -e "\n***\n*** Failed to start $SERVER\n***"
+    cat $SERVER_LOG
+    exit 1
+fi
+# The environment should be extracted
+curl -v -X POST localhost:8000/v2/repository/models/python_3_8/load
+touch -m models/python_3_8/1/model.py
+# The environment should not be re-extracted
+curl -v -X POST localhost:8000/v2/repository/models/python_3_8/load
+touch -m models/python_3_8/python_3_8_environment.tar.gz
```

Comment:
Maybe use the Python 3.6 env with the same name as Python 3.8 environment to make sure it is really using the new environment?